WordPress/src/main/java/org/wordpress/android/ui/people/PeopleListFragment.java

```diff
@@ -157,6 +157,15 @@ public int getItemCount() {
         }
 
         @Override
+        public long getItemId(int position) {
```

Comment:
In addition to this, the adapter's constructor should have `setHasStableIds(true);`